plugins/modules/pipx.py

```diff
@@ -57,7 +57,7 @@
     install_deps:
         description:
             - Include applications of dependent packages.
-            - Only used when I(state=install), I(state=latest), I(state=upgrade), or I(state=inject).
+            - Only used when I(state=install), I(state=latest), or I(state=inject).
```

Comment:
If I got it right, we should remove `latest` from here as well, no?

Reply:
It still works with state=latest.  Since state=latest invokes 'install' and then 'upgrade' tasks, 'install_deps' is still applicable to the install itself, just not the upgrade.

See https://github.com/ansible-collections/community.general/pull/6303/files#diff-ab43a68081a0292e6515548b437d0ff6b706fb86ccefa64d2549b40c66b82b84R308-R309

(just a little outside the default diff context)